kogitoq/kogito-common/src/main/java/life/genny/kogito/common/core/ProcessAnswers.java

```diff
@@ -104,17 +105,24 @@ public Boolean checkUniqueness(ProcessData processData) {
 		if (answers.isEmpty())
 			return acceptSubmission;
 
-		Answer answer = answers.get(answers.size() - 1);
-		String attributeCode = answer.getAttributeCode();
-
 		if (qwandaUtils.isDuplicate(definitions, null, processEntity, originalTarget)) {
 			String feedback = "Error: This value already exists and must be unique.";
 
-			String parentCode = processData.getQuestionCode();
-			String questionCode = answer.getCode();
+			for(Definition definition : definitions){
```

Comment:
We may need to review this in future, as there are some minor concerns for this error sending logic. For now it should be fine.